tools/mentors_test.yml

```diff
@@ -0,0 +1,153 @@
+- name: Adriana Zencke Zimmermann
```

Comment:
I think this file could be removed right?

Reply:
Yes, it was just for demonstration of how does it look like :-)